src/test/java/org/jpeek/MetricsTest.java

```diff
@@ -181,7 +181,11 @@
             new Object[] {"NoMethods", "CCM", Double.NaN},
             new Object[] {"WithoutAttributes", "CCM", Double.NaN},
             new Object[] {"OneMethodCreatesLambda", "CCM", Double.NaN},
-            new Object[] {"OneVoidMethodWithoutParams", "CCM", Double.NaN}
+            new Object[] {"OneVoidMethodWithoutParams", "CCM", Double.NaN},
+            new Object[] {"Foo", "CCM", 0.1667d},
+            new Object[] {"OverloadMethods", "CCM", 0.6d},
+            new Object[] {"TwoCommonAttributes", "CCM", Double.NaN},
+            new Object[] {"TwoCommonMethods", "CCM", 0.0238d}
         );
```

Comment:
@uurl `TwoCommonMethods` test for `CCM` metric was not asked in issue, why it has been added?